core/src/test/java/org/jdbi/v3/core/mapper/ImmutablesTest.java

```diff
@@ -182,4 +185,52 @@ public void testByteArray() {
                 .value())
             .containsExactly(value);
     }
+
+    @Value.Immutable
+    public interface DerivedProperty {
+        @Value.Default
+        default int foo() {
+            return 1;
+        }
+
+        @Value.Check
+        default void checkFoo() {
+            if (foo() == 999) {
+                throw new Boom();
+            }
+        }
+
+        @Value.Derived
+        @JdbiProperty(mappable = false)
```

Comment:
This is the new feature -- previously, any attempt to use `@Value.Derived` makes your type totally unmappable.